gcp_variant_transforms/transforms/merge_header_definitions.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beam combiner function for merging VCF file header definitions."""
+
+from collections import namedtuple
+from typing import Dict, List  # pylint: disable=unused-import
+
+import apache_beam as beam
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader  # pylint: disable=unused-import
+from gcp_variant_transforms.libs.vcf_field_conflict_resolver import VcfParserConstants
+
+Definition = namedtuple('Definition', [VcfParserConstants.NUM,
+                                       VcfParserConstants.TYPE])
+
+
+class VcfHeaderDefinitions(object):
+  """Container for header definitions."""
+
+  def __init__(self, vcf_header=None):
+    # type: (VcfHeader) -> None
+    """Initializes a VcfHeaderDefinitions object.
+
+    Creates two dictionaries (for infos and formats respectively) that map field
+    id to a dictionary which maps Definition to a list of file names.
```

Comment:
use double back quote for class name, i.e. ``Definition`` ([example](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/libs/bigquery_vcf_schema.py#L48)). Can you check if this is still how we reference classes? Thanks!

Reply:
Done. I checked some other classes, it seems that the double back quotes are the most commonly used one.